www/source/partials/docs/_using-hab-install-faq.html.md.erb

```diff
@@ -17,14 +17,7 @@ A: We've got you covered! The script we provide for doing curlbash installations
 
 A: Indeed they are divisive, we know, that's why we provide a few different ways for you to download. If you'd like to take a look at the script in advance of running it, you can find it in [the core habitat repo](https://github.com/habitat-sh/habitat/blob/master/components/hab/install.sh).
 
-Install Habitat via curlbash using the following command:
-`curl https://raw.githubusercontent.com/habitat-sh/habitat/master/components/hab/install.sh | sudo bash`
-
-If you're staunchly in the anti-curlbash camp you can get the latest packages from our [download page](https://habitat.sh/downloads) or via releases on [our bintray page](https://bintray.com/habitat)
-
-**Q: Er, how do I install habitat?**
-
-A: You'll need to download the `hab` package. Extract the binary to somewhere on your $PATH (or somewhere that you can execute the binary from a command line). If you're installing on a workstation we would strongly recommend running `hab setup` after youve extracted the binary. Read our [step by step docs here](/docs/using-habitat/#install-habitat).
+If you're staunchly in the anti-curlbash camp, you can get the latest packages from the [download links](/docs/install-habitat/#install-habitat) listed previously, or via releases on our [bintray page](https://bintray.com/habitat)
```

Comment:
Little thing, but missing a period. And should Bintray be capped?

Reply:
That actually brings up a good point about how Habitat itself is kind of inconsistently capitalized in this doc - older stuff seems to stick with lowercase while more recently we've been capitalizing it.

Reply:
Ah, yeah. I'm basically capitalizing that everywhere, unless it's a reference to the actual `hab` binary of course.